Documentation/cmdref/cilium-agent.md

```diff
@@ -74,6 +74,7 @@ cilium-agent [flags]
       --enable-ipv4                                   Enable IPv4 support (default true)
       --enable-ipv4-fragment-tracking                 Enable IPv4 fragments tracking for L4-based lookups (default true)
       --enable-ipv6                                   Enable IPv6 support (default true)
+      --enable-ipv6-ndp                               Enable IPv6 NDP support
```

Comment:
It's auto generated. I guess this means (default false).

Reply:
Can we add a minor help addition something like "network must be fully trusted".